src/trace/traced_process.cc

```diff
@@ -16,7 +16,7 @@
 
 using namespace std;
 
-int SYSCALL_ARG_REGS[] = { ORIG_RAX, RDI, RSI, RDX, R10, R8, R9 };
+int SYSCALL_ARG_REGS[] = { /* ORIG_RAX, */ RDI, RSI, RDX, R10, R8, R9 };
```

Comment:
The first one (`ORIG_RAX`) is the syscall number and the rest are the arguments. I just wanted to access to the arguments using this array, so I commented this one out!